docs/en/plugins/phar.md

```diff
@@ -6,11 +6,12 @@ Allows you to create a [Phar](http://php.net/manual/en/book.phar.php) archive fr
 Configuration
 -------------
 
+See also [Common Plugin Configuration Options](../plugin_common_options.md).
+
 ### Options
 
-* **directory**: `phar` output directory. Default: `%buildpath%`;
```

Comment:
Plugin like this pretty different `directory` purpose. In such cases it could be left.

Reply:
Another option is to use `directory` only as input directory and add new `output_path` line  like other plugins.

But this is idea for separate PR.

Reply:
@garas It is good idea for version `2.0.0` (because it is backward incompatible).

Reply:
We may add new option `output_path` as alternative in minor version `1.x` and deprecate `directory` for some plugins.